changelog

```diff
@@ -15,3 +15,12 @@ inappropriate conduct that falls outside of the scope of the CoC
 - Clarifies scope of applicability, specifically the 'representing the community' clause
 - Adds provisions against personal and political attacks
 - Notes that maintainers are responsible for clarification of questions of acceptable behavior
+
+2.0.0
+- Emphasizes 'community' over 'project' scope. It effectively merges the Community Covenant into the Contributor Covenant
+- Adds expectation about accepting responsibility and apologizing to those affected by mistakes, and learning from the experience to the list
+- Adds sexual attention and advances of any kind as unacceptable behavior example
+- Moves enforcement responsibilities from project maintainers to community leaders
+- Adds responsibility to communicate reasons for moderation decisions when removing, editing, or rejecting contributions not aligned to the Code of Conduct
+- Requires to respect privacy and security of the reporter of any incident, not just confidentiality
+- Standardizes how to enforce code of conduct in the case of Code of Conduct violations
```

Comment:
```suggestion
- Standardizes enforcement for Code of Conduct violations
```

Reply:
"Provides a template for code of conduct enforcement" is the language I would prefer.

Reply:
```suggestion
- Provides a template for code of conduct enforcement
```